<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QWidget" name="widgetBtm" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="CameraControl" name="widgetCameraB" native="true"/>
      </item>
      <item>
       <widget class="CameraControl" name="widgetCameraC" native="true"/>
      </item>
      <item>
       <widget class="CameraControl" name="widgetCameraD" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="DateTimeControl" name="widgetTime" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="CameraControl" name="widgetCameraA" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="WeatherControl" name="widgetWeather" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraControl</class>
   <extends>QWidget</extends>
   <header location="global">cameracontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DateTimeControl</class>
   <extends>QWidget</extends>
   <header location="global">datetimecontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeatherControl</class>
   <extends>QWidget</extends>
   <header location="global">weathercontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
